### ISSUE TITLE
Camera inputs update loop should be in its own struct/class function

### ISSUE THREAD

**Issue by ze413X** [rikhardsvensson/StortSpel-Grupp1#82]
So all:

```
/*
Toggle free camera mode
*/
HandleCamMode();

/*
Camera scroll
*/
HandleCamZoom(deltaTime);

/*
Camera rotation
*/
HandleCamRot();

/*
Camera move
*/
HandleCamMove(deltaTime);
```

Is centered and not typed in 4 different places.

**Comment by ze413X** [comment]
Add functionality in camera.h

**Comment by SebOnTheWeb** [comment]
Fixed with adding the camera handler functions to basestate. States then have to call that function to check for the camera input. This is for instance not done in menustate.

Fixed in 1e32397d5fbb597fe6a42ab05ce4440ef0a92b62
